test/app.test.ts

```diff
@@ -19,6 +19,23 @@ describe('app', () => {
     expect(res.body).toEqual({ url: '/' })
   })
 
+  it('can return a 204 response', async () => {
+    app.use('/api', () => null)
+    const res = await request.get('/api')
+
+    expect(res.statusCode).toBe(204)
+    expect(res.text).toEqual('')
+    expect(res.ok).toBeTruthy()
+  })
+
+  it('can return primitive values', async () => {
+    app.use('/number', () => 42)
+    expect(await request.get('/number').then(r => r.body)).toEqual(42)
+
+    app.use('/bool', () => true)
```

Comment:
Is `false` handling broken?

Reply:
nope, my bad!

Reply:
Happy to remove test if needed.

Reply:
Seems good. I tought `false` as test case has more coverage than true. We can test both too...

Reply:
Ahh. Good point.